crates/bevy_scene/src/bundle.rs

```diff
@@ -0,0 +1,94 @@
+use std::ops::{Deref, DerefMut};
+
+use bevy_asset::Handle;
+use bevy_ecs::{
+    bundle::Bundle,
+    change_detection::ResMut,
+    entity::Entity,
+    prelude::{Changed, Component},
+    system::{Commands, Query},
+};
+use bevy_transform::components::{GlobalTransform, Transform};
+
+use crate::{DynamicScene, InstanceId, Scene, SceneSpawner};
+
+/// [`InstanceId`] of a spawned scene. It can be used with the [`SceneSpawner`] to
+/// interact with the spawned scene.
+#[derive(Component)]
+pub struct SceneInstance(InstanceId);
+
+impl Deref for SceneInstance {
+    type Target = InstanceId;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl DerefMut for SceneInstance {
+    fn deref_mut(&mut self) -> &mut <Self as Deref>::Target {
+        &mut self.0
+    }
+}
+
+/// A component bundle for a [`Scene`] root.
+///
+/// Once the scene is spawned, the entity will have a [`SceneInstance`] component.
```

Comment:
The PR description says the scene will be added as a Child. Is that still accurate? If so, it should be documented here and on `DynamicSceneBundle`.

Reply:
That's what I meant by `root`. I tried to improve the doc 👍